schema/postgresql/v96/temporal/versioned/v1.1/cluster_metadata.sql

```diff
@@ -1,3 +1,3 @@
-ALTER TABLE cluster_metadata ADD data BYTEA NOT NULL;
-ALTER TABLE cluster_metadata ADD data_encoding VARCHAR(16) NOT NULL;
-ALTER TABLE cluster_metadata ADD version BIGINT DEFAULT 1 NOT NULL;
\ No newline at end of file
+ALTER TABLE cluster_metadata ADD data BYTEA NOT NULL DEFAULT '';
+ALTER TABLE cluster_metadata ADD data_encoding VARCHAR(16) NOT NULL DEFAULT 'Proto3';
```

Comment:
I think @vitarb didn't add these defaults intentionally.

Reply:
> I think @vitarb didn't add these defaults intentionally.

without it, it will be broken during schema upgrade

Reply:
sequence to break (postgresql):
1. checkout 1.0.0
2. create DB & schema
3. start server
4. checkout 1.1 or 1.2
5. upgrade schema -> break
```
2020/10/27 12:46:57 ---- Executing updates for version 1.1 ----
2020/10/27 12:46:57 ALTER TABLE cluster_metadata ADD data BYTEA NOT NULL;
2020/10/27 12:46:58 error executing CQL statement:pq: null value in column "data" violates not-null constraint
```